tools/data_collection.py

```diff
@@ -0,0 +1,110 @@
+import os
+import sys
+import asyncio
+import json
+
+from dotenv import load_dotenv
+
+#import the app.py module to gain access to the methods to construct payloads and
+#call the API through the sdk
+
+# Add parent directory to sys.path
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
+
+import app
+
+#function to enable loading of the .env file into the global variables of the app.py module
+
+def load_env_into_module(module_name, prefix=''):
+    load_dotenv()
+    module = __import__(module_name)
+    for key, value in os.environ.items():
+        if key.startswith(prefix):
+            setattr(module, key[len(prefix):], value)
+
+load_env_into_module("app")
+
+#some settings required in app.py
+
+app.SHOULD_STREAM = False
+app.SHOULD_USE_DATA = app.should_use_data()
+
+#format:
+"""
+[
+  {
+    "qa_pairs":[{"question":"...", "answer":"..."}]
+  }
+]
+"""
+
+generated_data_path = r"path/to/qa_input_file.json"
+
+with open(generated_data_path, 'r') as file:
+    data = json.load(file)
+
+evaluation_data_set = []
+
+for qa_pairs_obj in data[:1]:
```

Comment:
why only the first obj? maybe the format should just be
{
  "qa_pairs": [{...}, {...}]
}
If we don't want to take multiple "qa_pairs" objects anyway.

Reply:
accidentally undeleted debug code.